Add section to explain how the manager certificates are generated. (#5461)
wazuh/wazuh-documentation: source/user-manual/reference/daemons/wazuh-authd.rst

.. warning::
  By default, there is no authentication or authorization involved in this transaction, so it is recommended that this daemon only be run when a new agent is being added.

> .. versionadded:: 5.0

Reviewer: This is not needed since the documentation has a version selector

> ``wazuh-authd`` is able to generate the manager certificates even if openssl is not installed on the system. The parameters of the certificate are specified in the CLI:

Reviewer: ```suggestion
``wazuh-authd`` is able to generate X.509 certificates even if OpenSSL is not installed on the system. The parameters of the certificate are specified in the CLI:
```

   .. code-block:: console

     # wazuh-authd -C 265 -B 2048 -K /var/ossec/etc/sslmanager.key -X /var/ossec/etc/sslmanager.cert -S "/C=US/ST=California/CN=wazuh/"

> If one of the parameters related with the certificate generation is missing, an error will be triggered and the certificate generation will be aborted.

Reviewer: ```suggestion
If any of the parameters related to the certificate generation is missing, an error will be triggered and the certificates are not generated.
```

> 

Reviewer: Include a brief explanation of what these certificates are for in Wazuh.

+------------------+-------------+-----------------------------------------------------------------------------------------+
| **-L**           | Force insertion even though agent limit has been reached.                                             |
+------------------+-------------------------------------------------------------------------------------------------------+
| **-C**           | Specify the manager certificate validity in days.                                                     |
+------------------+-------------------------------------------------------------------------------------------------------+
| **-B**           | Set the manager certificate key size in bits.                                                         |
+------------------+-------------------------------------------------------------------------------------------------------+

> | **-K**           | Path to store the manager certificate key.                                                            |

Reviewer: Let's avoid talking about `manager certificate`. This tool can be used to generate an X.509 certificate for any purpose, even the default one is to create the certs for the manager.

+------------------+-------------------------------------------------------------------------------------------------------+
| **-X**           | Path to store the manager certificate.                                                                |
+------------------+-------------------------------------------------------------------------------------------------------+

> | **-S**           | Subject of the manager certificate.                                                                   |

Reviewer: From the OpenSSL documentation, we could specify the required format:
```
The arg must be formatted as /type0=value0/type1=value1/type2=....  Keyword characters may be escaped by \ (backslash), and whitespace is retained.  Empty values
           are permitted, but the corresponding type will not be included in the request.
```
